FirebaseCore/Tests/Unit/Swift/SwiftTestingUtilities/ExceptionCatcher.h

```diff
@@ -12,11 +12,17 @@
 // See the License for the specific language governing permissions and
```

Comment:
I'm not sure why it says I renamed `FIRAppTests.swift` to `ExceptionCatcher.h`. I used `git mv` to keep the `FIRAppTests.swift` history when I renamed it to `FirebaseAppTests.swift`. I kept trying to correct it but couldn't get it right. I read that git can diff based on the contents and I think that is part of the problem because every time I got the history corrected, it would revert back when I added the contents of each file back in :(